src/sentry/api/endpoints/release_file_details.py

```diff
@@ -70,6 +72,13 @@ class ReleaseFileDetailsEndpoint(ProjectEndpoint):
     doc_section = DocSection.RELEASES
     permission_classes = (ProjectReleasePermission,)
 
+    def download(self, releasefile):
+        file = releasefile.file.getfile()
+        response = HttpResponse(FileWrapper(file), mimetype='application/octet-stream')
```

Comment:
We should also extract the `mimetype` from `file.headers.get('content-type', 'application/octet-stream')` since we typically have this information.